### ISSUE TITLE
capitalized-comments causes eslint to terminate with error on comment with just a space

### ISSUE THREAD

**Issue by Trott** [eslint/eslint#7663]
**Tell us about your environment**

* **ESLint Version: 3.11.0
* **Node Version: 7.2.0
* **npm Version: 3.10.9

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

No configuration file.
```




```

**What did you do? Please include the actual source code causing the issue.**

With this file:

```js
// 
```

Note that there is a single space after the `//`.

I ran: `eslint --rule 'capitalized-comments: 2'`

**What did you expect to happen?**
Expected lint to be successful.

**What actually happened? Please include the actual, raw output from ESLint.**
```console
Cannot read property 'toLocaleLowerCase' of undefined
TypeError: Cannot read property 'toLocaleLowerCase' of undefined
    at isCommentValid (/Users/trott/.nvm/versions/node/v7.2.0/lib/node_modules/eslint/lib/rules/capitalized-comments.js:242:64)
    at processComment (/Users/trott/.nvm/versions/node/v7.2.0/lib/node_modules/eslint/lib/rules/capitalized-comments.js:262:32)
    at Array.forEach (native)
    at EventEmitter.Program (/Users/trott/.nvm/versions/node/v7.2.0/lib/node_modules/eslint/lib/rules/capitalized-comments.js:287:26)
    at emitOne (events.js:96:13)
    at EventEmitter.emit (events.js:188:7)
    at NodeEventGenerator.enterNode (/Users/trott/.nvm/versions/node/v7.2.0/lib/node_modules/eslint/lib/util/node-event-generator.js:40:22)
    at CodePathAnalyzer.enterNode (/Users/trott/.nvm/versions/node/v7.2.0/lib/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:608:23)
    at CommentEventGenerator.enterNode (/Users/trott/.nvm/versions/node/v7.2.0/lib/node_modules/eslint/lib/util/comment-event-generator.js:97:23)
    at Controller.enter (/Users/trott/.nvm/versions/node/v7.2.0/lib/node_modules/eslint/lib/eslint.js:925:36)

```

**Comment by not-an-aardvark** [comment]
I can reproduce this, thanks for reporting.
